packages/playground/src/components/node_details_cards/card_details.vue

```diff
@@ -79,7 +79,7 @@ export default {
 
     icon: {
       type: String,
-      required: true,
+      required: false,
```

Comment:
is the icon prop optional? if yes, should we handle undefined cases on the template?

Reply:
what do u mean of undefined cases ? It's called one time in <h4> (card title) and could be added or not.

Reply:
Did you try to not add it? Does it worked fine without any error?

Reply:
- with icon 
![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/54944491/c28e6cef-c8d2-4d5d-b683-85ff277ff008)

- without icon
![Screenshot from 2023-12-18 12-18-14](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/54944491/230bd620-a4ed-4f4b-a04e-3b0282e7d501)